File: LV0/배열 만들기 2.js
function solution(l, r) {
  let result = [];
  for (let num = l; num <= r; num++) {
    let strNum = String(num);
    if (strNum.split("").every((char) => char === "0" || char === "5")) {
      result.push(num);
    }
  }
  if (result.length === 0) {
    return [-1];
  }

  return result;
}

File: LV0/배열 만들기 2.java
class Solution {

  public int[] solution(int l, int r) {
    List<Integer> result = new ArrayList<>();
    for (int num = l; num <= r; num++) {
      String strNum = String.valueOf(num);
      if (strNum.chars().allMatch(c -> c == '0' || c == '5')) {
        result.add(num);
      }
    }
    if (result.size() == 0) {
      return new int[] { -1 };
    }

    return result.stream().mapToInt(i -> i).toArray();
  }
}
